specification/search/data-plane/Microsoft.Azure.Search.Data/preview/2017-11-11-preview/searchindex.json

```diff
@@ -79,13 +82,27 @@
             "$ref": "#/parameters/ApiVersionParameter"
           },
           {
-            "$ref": "#/parameters/AutocompleteModeParameter"
+            "name": "autocompleteMode",
+            "in": "query",
+            "required": true,      
+            "type": "string",
+            "enum": [
+              "oneTerm",
+              "twoTerms",
+              "oneTermWithContext"
+            ],
+            "x-ms-enum": { 
+             "name": "AutocompleteMode",
+             "modelAsString": false
+            },
+            "x-ms-parameter-location": "method",
+            "description": "Autocomplete mode."
           },
           {
             "name":"search",
```

Comment:
Maybe the `search` parameter should be first. I think it's the first parameter for `Search` and `Suggest`. I would argue it's the most important parameter.